sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlobBeginCopyOptions.java

```diff
@@ -108,17 +108,18 @@ public BlobBeginCopyOptions setRehydratePriority(RehydratePriority rehydratePrio
     }
 
     /**
-     * @return {@link RequestConditions} for the source.
+     * @return {@link BlobBeginCopySourceRequestConditions} for the source.
      */
-    public RequestConditions getSourceRequestConditions() {
+    public BlobBeginCopySourceRequestConditions getSourceRequestConditions() {
         return sourceRequestConditions;
     }
 
     /**
-     * @param sourceRequestConditions {@link RequestConditions} for the source.
+     * @param sourceRequestConditions {@link BlobBeginCopySourceRequestConditions} for the source.
      * @return The updated options.
      */
-    public BlobBeginCopyOptions setSourceRequestConditions(RequestConditions sourceRequestConditions) {
+    public BlobBeginCopyOptions setSourceRequestConditions(
```

Comment:
If this has shipped in the past this will need to be a new override

Reply:
No, this is a new type